packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/object/controllers/ObjectRestController.java

```diff
@@ -55,6 +61,14 @@ public ObjectRestController(IEditingContextDispatcher editingContextDispatcher)
     }
 
     @Operation(description = "Get all the elements in a given project at the given commit.")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "OK", content = {
+            @Content(mediaType = "application/json", examples = {
+                @ExampleObject(value = "[{\"@id\":\"9f2e43d4-2f7e-49f1-816a-a4e0d3d1f370\", \"attribute1\":\"hello\"}, {\"@id\":\"bab1f7be-82e0-4d14-bc60-b12a60c46f2f\", \"attribute1\":\"bye\"}]")
+            })
```

Comment:
I don't really mind but is there a specific reason why you have here an example and on other APIs returning one or more objects you have instead a schema with a reference to `Object.class`?

Reply:
`Object.class` returns 
```
[{}]
```
as example.

Reply:
Yes I saw that, it's just that the API to return all the elements will display this:

<img width="1274" alt="Capture d’écran 2025-01-13 à 11 01 16" src="https://github.com/user-attachments/assets/c97de4c6-d019-400b-9d16-06e1ddc4c35b" />


While the API to return one of those elements will only display that:

<img width="1276" alt="Capture d’écran 2025-01-13 à 11 01 28" src="https://github.com/user-attachments/assets/fcdab787-94fd-4bca-b400-4c987f5b56a0" />


One could expect that in the second use case the example would display the same level of details. I'm not against merging things like that, I just want to be sure that it's not an issue for your consumers in SysON. No-one is really using this in Sirius Web today.

Reply:
May be I have missed one case?

Reply:
Yes I have :)

Reply:
I will fix that soon

Reply:
Done